.gitignore

```diff
@@ -30,3 +30,4 @@
 *.exe
 *.out
 *.app
+6_EOGDinoGame/venv/
```

Comment:
This should not be specific to a project folder, please make it generic.

Reply:
I have made the updates regarding this. Please review.